physicalplan/aggregate/hashaggregate.go

```diff
@@ -30,32 +30,33 @@ type aggregate struct {
 	tables         []aggregateTable
 	series         []labels.Labels
 	newAccumulator newAccumulatorFunc
-
-	stepsBatch int
-	workers    worker.Group
+	param          interface{}
```

Comment:
I think we should be able to use a more specific type here, at least `parser.Item` or something similar.

Reply:
I will look